generators/cloud/templates/jenkins/jobs-optional/deploy-env/config.xml

```diff
@@ -90,11 +90,19 @@ trap &apos;complete $?&apos; EXIT
 teardown
 
 # Set up and install site
-FLAGS=&apos;&apos;
+docker-compose -f docker-compose.yml up -d
+docker-compose run --rm cli "NPM_CONFIG_LOGLEVEL=error npm install"
+docker-compose run --rm grunt --timer
+
 if [ $UPDATE = &quot;true&quot; ]; then
-  FLAGS=&quot;$FLAGS --update&quot;
+  docker-compose run --rm grunt update
+else
+  # Remove --no-db-load if each deploy should re-load a database from /opt/backups/latest.sql.gz.
+  # The location and name of this file can be changed in Gruntconfig.json.
+  docker-compose run --rm grunt install --no-db-load
```

Comment:
I think it would be nice if we change all instances of `--no-db-load` to something like `<% if (!latestDB) { %> --no-db-load<% } -%>`.

Reply:
This will always run a site install when deploying an env. Seems like we may want this to be a job parameter?

Reply:
@scottalan @febbraro There is a job parameter to update the environment against the database in place. Otherwise it does a clean install. My experience is that if you want a given job to load from an existing source database, that's usually more of a permanent "switch" for a given job and grounds to simply edit the config.xml.

Are there grounds for this structure to deploy a new environment without installing Drupal or loading a database?